examples/08.cancelOrder.js

```diff
@@ -19,25 +21,27 @@ const web3 = new Web3(provider)
 provider.engine.stop()
 
 const dvfConfig = {
-  api: envVars.API_URL
+  api: envVars.API_URL,
+  dataApi: envVars.DATA_API_URL
   // Add more variables to override default values
 }
 
 ;(async () => {
   const dvf = await DVF(web3, dvfConfig)
 
+  const P = require('aigle')
   let orderId
-  const orders = await dvf.getOrders('ETH:USDT')
+  const orders = await dvf.getOrders()
 
   console.log('orders', orders)
 
   if (orders.length == 0) {
     console.log('submitting new order')
-    
-    // Submit an order to buy 0.5 Eth at a rate of 200 USDT for 1 Eth
+
+    // Submit an order to sell 0.1 ETH at a the price of 5000 USDT per ETH
     const symbol = 'ETH:USDT'
-    const amount = 0.5
-    const price = 200
+    const amount = -0.1
+    const price = 5000
```

Comment:
How about we get the current price like we are doing here
https://github.com/DeversiFi/dvf-client-js/blob/ag/examples/improve-tests/examples/05.submitOrder.js#L32
and increase like 20%?

Reply:
I guess we could, although I'd increase it by a factor of 10 or something, just to make sure it will definitely not get picked up before we manage to cancel it